File: verve patch/base/Applications/CSymphony_0_1/csSamples/SourceOut/CodeMethodReturnStatement.cpp
#pragma once 

#using <mscorlib.dll>

namespace MyNameSpace
{
    
    using namespace System;
    public __gc class MyClass;
    
    
    public __gc class MyClass
    {
        
        public: virtual System::String *  ReturnString(System::String *  text);
    };
    
    
    inline System::String *  MyClass::ReturnString(System::String *  text)
    {
        return text;
    };
}

File: verve patch/base/Applications/CSymphony_0_1/csSamples/CodeMethodReturnStatement.cs
using System;
using System.CodeDom;
using System.CodeDom.Compiler;
using System.Collections;
using CSymphony.Lib;

namespace CSymphony.CSy
{
	public class CSyUnit : GraphUnit
	{
		public override void Build()
		{
			ns("MyNameSpace");
			declClass(TypePublic, "MyClass");

			ArgRef argText = "text";
			declMethod(MemPublic, "System.String","ReturnString", "System.String", argText);
			rtn(argText);

		}
	}
}
